server/controllers/users.js

```diff
@@ -28,6 +28,12 @@ class User {
       .fetch({ withRelated })
       .then(user => Promise.resolve(user ? user.toJSON() : null));
   }
+
+  static getAll(identifier, withRelated) {
```

Comment:
missing tests (/test/unit/controllers)

Reply:
Well, you never wrote tests for `get`... 😛

I guess I'll be nice and write them both. 😝